drivers/hmis/lib/form_data/allegheny/occurrence_point_forms/move_in_date.json

```diff
@@ -28,15 +28,16 @@
       ]
     },
     {
-      "link_id": "3.20.custom.address",
-      "type": "TEXT",
+      "type": "OBJECT",
+      "component": "ADDRESS",
+      "repeats": false,
       "required": true,
-      "text": "Move-in Address",
-      "readonly_text": "Address",
+      "link_id": "addresses",
       "mapping": {
         "record_type": "ENROLLMENT",
-        "custom_field_key": "move_in_date_address"
+        "field_name": "addresses"
```

Comment:
This feels brittle, as soon as we start supporting tying other addresses to Enrollments, this will break.

I think it would be ideal if there is a field `moveInAddress` that resolves the specific CustomClientAddress that is for Move-In at this Enrollment.. see my other comment about that